README.md

```diff
@@ -54,12 +54,30 @@ Example:
 $ stix-shifter translate qradar query {} "[ipv4-addr:value = '127.0.0.1']" {}
 ```
 
-**Note:** In order to build `stix-shifter` packages from source follow the below prerequisite steps:
+In order to build `stix-shifter` packages from source follow the below prerequisite steps:
    1. Go to the stix-shifter parent directory
-   2. Generate latest requirements.txt: `python3 generate_requirements.py`
-   3. Install the dependencies in your python 3 environment: `pip install -r requirements.txt` 
-   4. Alternatively you can create a Python 3 virtual environemnt:
-       `virtualenv -p python3 virtualenv && source virtualenv/bin/activate && pip install -r requirements-dev.txt`
+   2. Optionally, you can create a Python 3 virtual environemnt:
+       `virtualenv -p python3 virtualenv && source virtualenv/bin/activate`
+   3. Run setup: `python3 setup.py install`
+
+
+### Running From the Source
+
+You may also use `python3 main.py` local script. All the options are the same as for the *"As A Command Line Utility"* usage above.
```

Comment:
You may also use `python3 main.py` local script. All the options are the same as *"As a command line utility"* usage above.